### ISSUE TITLE
Responsive project layout

### ISSUE THREAD

**Issue by amyjko** [wordplaydev/wordplay#810]
## Problem

The project view, while functional for laptop-sized screens and larger at smaller font sizes, fails in several situations:

* When the font size is increased in the browser, there's insufficient space for the full layout
* On a tablet or smartphone screen, even at small font sizes, the application has so much occlusion, it becomes non-functional

Here are a few examples:

![Image](https://github.com/user-attachments/assets/371b307c-23af-4084-ad0a-14e265a7df37)

![Image](https://github.com/user-attachments/assets/6a7d39ab-2946-4bfd-b9a0-dc727358f8a6)

There were two issues that had some solid design ideas for how to address this: #368 and #409. Both were designed independently, and not really reconciled, so this issue brings those designs together in one place to facilitate implementation.

## Design

The core design idea is to have an overarching design of an automatically tiling layout of windows. There are three cutoff points:

* At 992px, we have the current layout functionality.
* At 768px, we permit up to two tiles open at any given time, with new tiles replacing the oldest existing tile open opening. If one tile is open, it is full screen. The split is along the dominant axis.
* At 544px, only one tile can be open at a time and it is always full screen

Additionally, the editor's annotation view moves to be be stacked vertically in the reading order when the vertical axis is dominant, so that conflicts are readable on smart phones when the editor is open.
